cmd/minishift/cmd/root.go

```diff
@@ -70,7 +60,9 @@ var viperWhiteList = []string{
 	"log_dir",
 }
 
-var hasEnabledExperimental bool = false
+var (
+	hasEnabledExperimental bool = false
```

Comment:
if you group var this way, then I would argue RootCmd should be in there as well